pages/getting-started/install-memgraph/docker-compose.mdx

```diff
@@ -274,6 +274,47 @@ connect port number, e.g. `QUICK_CONNECT_MG_PORT: 7688`
 
 ---
 
+## Adding a Health Check to Docker Compose for Memgraph
```

Comment:
Could the subtitle be just **Adding a Health Check** since this is already under a Docker Compose page just to shorten it?